sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - We are now targeting the service's v3.1-preview.1 API as the default. If you would like to still use version v3.0 of the service,
 pass in `v3.0` to the kwarg `api_version` when creating your TextAnalyticsClient
 - We have added an API `recognize_pii_entities` which returns entities containing personal information for a batch of documents. Only available for API version v3.1-preview.1 and up.
+  - In API version v3.1-preview.2 and up, the redacted text of the document is returned on the top-level result object through property `redacted_text`
```

Comment:
This is only for PII, right? should we specify the top-level result object type?

Reply:
I can, I didn't think it was necessary, since this entry is actually showing up indented under the previous entry about the introduction of `recognize_pii_entities`. I'll add in the top level result object though, since there's one other docstring comment and I might as well fix both